src/pages/ContentScripts/features/repo-networks/base64.ts

```diff
@@ -0,0 +1,2 @@
Q9ImltYWdlL3BuZyIgcGhvdG9zaG9wOkNvbG9yTW9kZT0iMyIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDpkOTE2M2Q5Yy1lNGRjLTRlMGMtOTgzZC1kZDM1OWRlOTk5YjQiIHhtcE1NOkRvY3VtZW50SUQ9ImFkb2JlOmRvY2lkOnBob3Rvc2hvcDpjYWVmMTlmZi1jMzEzLTFjNDYtODgxOC0wYWM4NTQ3ZmQxZTYiIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDpmNjZlZWE1OS0zOTIwLTQwZjQtOTI3NC1kOWVkNWJmMDNhNTgiPiA8eG1wTU06SGlzdG9yeT4gPHJkZjpTZXE+IDxyZGY6bGkgc3RFdnQ6YWN0aW9uPSJjcmVhdGVkIiBzdEV2dDppbnN0YW5jZUlEPSJ4bXAuaWlkOmY2NmVlYTU5LTM5MjAtNDBmNC05Mjc0LWQ5ZWQ1YmYwM2E1OCIgc3RFdnQ6d2hlbj0iMjAyNC0xMC0xNlQyMDoyMzowNSswODowMCIgc3RFdnQ6c29mdHdhcmVBZ2VudD0iQWRvYmUgUGhvdG9zaG9wIDI0LjAgKE1hY2ludG9zaCkiLz4gPHJkZjpsaSBzdEV2dDphY3Rpb249ImNvbnZlcnRlZCIgc3RFdnQ6cGFyYW1ldGVycz0iZnJvbSBhcHBsaWNhdGlvbi92bmQuYWRvYmUucGhvdG9zaG9wIHRvIGltYWdlL3BuZyIvPiA8cmRmOmxpIHN0RXZ0OmFjdGlvbj0ic2F2ZWQiIHN0RXZ0Omluc3RhbmNlSUQ9InhtcC5paWQ6ZDkxNjNkOWMtZTRkYy00ZTBjLTk4M2QtZGQzNTlkZTk5OWI0IiBzdEV2dDp3aGVuPSIyMDI0LTEwLTE3VDEyOjE2OjAxKzA4OjAwIiBzdEV2dDpzb2Z0d2FyZUFnZW50PSJBZG9iZSBQaG90b3Nob3AgMjQuMCAoTWFjaW50b3NoKSIgc3RFdnQ6Y2hhbmdlZD0iLyIvPiA8L3JkZjpTZXE+IDwveG1wTU06SGlzdG9yeT4gPC9yZGY6RGVzY3JpcHRpb24+IDwvcmRmOlJERj4gPC94OnhtcG1ldGE+IDw/eHBhY2tldCBlbmQ9InIiPz6yUdnIAAAFRUlEQVR42u2d0Y2kMAyGKYESKIHneUoJlIB0DaSEdEAJlEAJlEAJlBDpGuCYW89d7nbYBZLYTmIkS/uy2g0fsRP7j1Nt21aJ5WXyEgSqmEAVE6hiAlVMoApUsdKgpvA8fvxsdxt2m3dbd9vg52k3vVudwjgE6gfMBuBt35jdzXCHWzzUHVAPsLYLtnAGWzRUALrdNLZgi4UKLtd6QP0dbwUqL6ijJ9CXdQKVB1AVCOgGq+RaoNJDXQJCfdogUGmBmsBAX6YEKl1yYYtklosbLgbq84U7WaJYNgtUXKhTZKBs4msRUCPG0SPrBSrfrFGSYLOGSgiUFGy2UBkAfZkRqGFWuSMToJtTk60F6v3ZuTID6u5jtUA9n8cdAlRcsGyFj68uHirAUyArGU4qFTibhTDxBNxkBRVioIbkwMIk/h1ZbMgvPZT5wr6d5aRQ4Z+0xAANeID6wkJMwe9RfoTj0QwngQovhuqFLCFjGoylJxqPfbcXRodKCHSMXR6DGTxSJzkooM4ELrYlKPFhj7MjgRpYRnLGNZHqh55/H3HN8Keeiw11zjGDcyLmYo1bo0KFwRVRzzyAO2AsArGhqtzrmByKDNhQdclAEcEqTKimdKBIYFGhdiXFUMIYiwo1hjxzrhJ+YqyKKbY06yNDna3ndscmu/qNoOrrqgyewGFJU0AN9WXOVUZPIDdMk1GCAYTY2rSZQQ1xVpYm9+sMwie2jlWGj6cbHjjUU332aarK9IGsm/Xdo5MpH27WVJcq8wfWHWeEdLyUDx6ztUd4qS3EfVcXpAlqsjW4ZAMAJ0ej1HDWKF2NrXXEl3hGM7ymkJKkhqqpXS/MiKvHHCfpo3QM9UqN1UQCungI2GqB+v7FTlSr3gAHkSeB6ueC6wgxNMuyHweo7dnqQ2Coa66FBRZnabBzvSc+JAt7RXNSxzsJ1OsJ7dBQ9TdA2xveRAtUWqjmaow8OWNbgcoTqvJIuLPoUyhQz8/UIZX9KxeollFM/QTlRnF/KhoqFIi5rX4XcLfKo4/ESDVjOUA9kwSwjPep7FwxB6hj4hkldqfvOEC1Ced+r57EU9lDvajL4Val8XHJOnRHFk5QR2pZ6M16akjAJnTSglLOcue8arRFB5NuaZe7xXCDahLSKP1vk9NbaYm8yJodbdIp2JQtd2xOakLYb3eQeVoR3HbPDWqfuzofAMdehK3vVtSi0I8PVyN0aOlJoZZ4lgbidGyX3JFALfnUG8KemKyPkn4UfD4VAayhgLo8Cj9JHuH0+D8LJwqo0vPhb4yNBbbNoTlWqt1ZYnUb7XLpeNYnBjRm2c/k1MauF6AfUtXUY2pqvQkxmk6qlFe/ycRY5AuQ6pT3qan0+20QC/Fzyhkl6cz9heSHIvfbERSeW2SYFD3055RLbz63XbQIMEciWUzNQfnQE100lNO9NG+1xdTCs4b4WkzfG6Qo75uz8D/U7HS/zovqHA2Qe7+aJQB9ZNQ3Py6gj9JffYQp3crYOjNkZHxPqs/HNDie47bwO+n7U8F994leuznH6qSWzU3HDOLzlTjYxHwXOd5Jrh48bz0esLJc2UFFko1cnZ0Kc/xZQmUEtszzqRmDJev9kDVUQrCkzTyyh0pQHSJvu1MEVAQF32G3NIGavj6IhZymKKiRdUJsmk4WBzWSVmp9MDotUCrUJnB8VZzGVyTUwPGVXcv1YqEC2DnAarcRqLyg+iYmeo7jKhqqB1jL+TRA8VAdsObk4ml+MG9PIFA/w9WgA1qc7cpLatKmMA40qGK8TV6CQBUTqGICVUygiglUgSqWsP0CEkgZNgQoviEAAAAASUVORK5CYII=';
```

Comment:
How large is this image? We can convert to smaller image with less pixels to reduce the size of this string.

Reply:
Yeah, I hadn't thought about it before. The original image was 117x117 px, I have now converted it to 36x36 px. This should reduce the size of the base64 string accordingly.